code/algorithm_applications/src/binary_search/peak_element_in_a_mountain_array/peak.dSYM/Contents/Resources/Relocations/x86_64/peak.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Remove binary and non-code files.

Only keep code files peak.c, cpp, py.

Reply:
done @AdiChat